pre-docs/writing_tasks.md

```diff
@@ -708,6 +708,51 @@ else
 fi
 ```
 
+If you intend to use this task with PE and assign RBAC permissions for it make
+sure the script safely handles parameters or validate them to prevent shell
+injection vulnerabilities.
+
+### Wrapping an existing script
+
+If the script is not already installed on targets and you do not want to edit
+it, for example if it's a script someone else maintains, you can wrap the
+script as is with it a small task.
```

Comment:
as is with it?

Reply:
`it` doesn't belong. 

I think `you can wrap the script in a small task without modifying it.` is clearer though